lib/ansible/module_utils/common/respawn.py

```diff
@@ -48,7 +50,7 @@ def respawn_module(interpreter_path) -> t.NoReturn:
     sys.exit(rc)  # pylint: disable=ansible-bad-function
 
 
-def probe_interpreters_for_module(interpreter_paths, module_name):
+def probe_interpreters_for_module(interpreter_paths: list[str], module_name: str | None = None, *module_names: str | None) -> str | None:
```

Comment:
I don't think `*module_names` is a good idea. I would use `*, module_names`, to make sure that `module_names` must always be a keyword argument:
```suggestion
def probe_interpreters_for_module(interpreter_paths: list[str], module_name: str | None = None, *, module_names: tuple[str, ...] | list[str] | None = None) -> str | None:
```

Reply:
ok,